testsuite/features/support/env.rb

```diff
@@ -64,6 +64,7 @@
 $is_using_scc_repositories = (ENV.fetch('IS_USING_SCC_REPOSITORIES', 'False') != 'False')
 $catch_timeout_message = (ENV.fetch('CATCH_TIMEOUT_MESSAGE', 'False') == 'True')
 $beta_enabled = (ENV.fetch('BETA_ENABLED', 'False') == 'True')
+$api_protocol = ENV.fetch('API_PROTOCOL', nil) if ENV['API_PROTOCOL'] # force the API protocol to be used. You can use 'http' or 'xmlrpc'
```

Comment:
I would set this to one of the two by default instead of nil. To avoid cases where it's empty and will cause problems down the line. Probably xmlrpc as it was before. It would allow the possibility to use http if we want without changing the current setup

Reply:
When this is `nil` it will fall into the `else` section of the `case` statement.
On such case, it will decide which protocol it will use depending the product version, as it was behaving before adding this variable to force the protocol.

The root cause of this PR is that some commits ago, we had defined that if `$debug_mode` was true, then we use XMLRPC API (no SSL certificates needed), and that allowed me to run my tests from my IDE without worry on SSL certificates. We removed this option, because at some point Dominik need to debug a test case using HTTP API with SSL.

To keep being able to debug on HTTP API, but please me to work without SSL certificates in my local environment... I added this variable to force the protocol, if the variable is defined.

Reply:
ok thanks for the explanation :+1: